tchannel/testing/vcr/patch.py

```diff
@@ -103,6 +103,7 @@ def send(self, arg1, arg2, arg3,
             vcr_response_future = tchannel.thrift(
                 proxy.VCRProxy.send(vcr_request),
                 hostport=self.vcr_hostport,
+                timeout=1.0,
```

Comment:
Oh, you're right. If a recording does not exist, a real request will be made so we should use the timeout from the original request + some delay for the proxy request.